crates/uv/src/commands/python/list.rs

```diff
@@ -78,6 +78,8 @@ pub(crate) async fn list(
             PythonListKinds::Installed => None,
             PythonListKinds::Downloads => Some(if all_platforms {
                 base_download_request
+            } else if all_arches {
+                base_download_request.fill()?.with_any_arch()
```

Comment:
This needs to be rebased and updated using fill_platform, now that #12915 has been merged to main.